extension/js/content_scripts/webmail/gmail-element-replacer.ts

```diff
@@ -529,17 +518,13 @@ export class GmailElementReplacer implements WebmailElementReplacer {
     const newReplyBoxes = $('div.nr.tMHS5d, td.amr > div.nr, div.gA td.I5').not('.reply_message_evaluated').filter(':visible').get();
     if (newReplyBoxes.length) {
       // cache for subseqent loop runs
-      const { drafts_reply } = await AcctStore.get(this.acctEmail, ['drafts_reply']);
       const convoRootEl = this.getGonvoRootEl(newReplyBoxes[0]);
       const replyParams = this.getLastMsgReplyParams(convoRootEl!);
-      const threadId = this.determineThreadId(convoRootEl!);
```

Comment:
I'm assuming the `threadId` is now retrieved from the API based on `replyParams.replyMsgId`?

Need to do throrough testing later, by sending emails between two accounts, to make sure they get correctly threaded on both ends.

Reply:
`threadId` isn't needed here anymore because the reply would be loaded automatically. Secure replies are threaded now after https://github.com/FlowCrypt/flowcrypt-browser/pull/3310/commits/45745be54b0585db215f4a7fff1b91a882ef368b